### PR TITLE
Add postgresql  plugin agent sql pararm show

### PR DESCRIPTION
**Pull request by aderm**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [X] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by aderm** [comment]
Add postgresql plugin agent sql pararm show

**Comment by aderm** [comment]
what is problem? @wu-sheng

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java [hunk 1]
```diff
@@ -263,7 +263,7 @@
              * If set to true, the parameters of the sql (typically {@link java.sql.PreparedStatement}) would be
              * collected.
              */
-            public static boolean TRACE_SQL_PARAMETERS = false;
```
Comment (JaredTan95) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java]: Let's keep it off by default.

**Comment by aderm** [comment]
can open it ?

**Comment by JaredTan95** [comment]
BTW, the submodule UI update should not be committed in one pull request.

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java (hunk 1 above)
Comment (aderm) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java]: add postgresql config in config file?

**Comment by aderm** [comment]
> BTW, the submodule UI update should not be committed in one pull request.

why? don't understand this ?

**Comment by aderm** [comment]
@JaredTan95  I do not change submodule UI, its auto change the file time, how to repair this ?

**Comment by wu-sheng** [comment]
If you don't know how to reset submodule to current, you have to recreate the branch from upstream master.

Also, as you are changing postgre plugin, we request you to submit another pr to move existing postgre plugin test into current one before we merge this. Like #3603 .

Than, you need to add this feature to plugin test case.(That is why I asked you you to do this)

**Comment by wu-sheng** [comment]
Submodule is changed by yourself, there should be your local merge issue.

**Comment by wu-sheng** [comment]
Why close this?

**Comment by aderm** [comment]
rebase master, find PR closed

**Pull request closed without merging**